### ISSUE TITLE
Query failed unexpectedly

### ISSUE THREAD

**Issue by GoogleCodeExporter** [runt18/google-bigquery#73]
```
Run query from BQ Web UI

Job ID: job_wjU2ZrNHThVtBpeKFPmoww-N63M

Did try multiple times over last two days

Result:

Query Failed
Error: Unexpected. Please try again.
Job ID: job_wjU2ZrNHThVtBpeKFPmoww-N63M

This query scan data only for 1 hour - so it should not be data size issue or 
performance (it just one simple REGEX). In the past we have similar issue when 
it was some problem with files google servers.


```

Original issue reported on code.google.com by `aam...@motorola.com` on 6 Mar 2014 at 5:00

**Comment by GoogleCodeExporter** [comment]
```
I'm having the same problem. I'm making queries to do quantiles calculation and 
all of a sudden I receive 500/502 error. Everything used to work perfectly fine 
until recently. This problem is preventing me from delivering new features to 
clients. Please help me asap. Thank you in advance.
```

Original comment by `system.u...@demandstreet.com` on 24 Mar 2014 at 1:09

**Comment by GoogleCodeExporter** [comment]
```
Regarding the original bug report, it looks like you've hit a bug in the query 
engine that causes some query errors to be incorrectly reported as internal 
errors when using allowLargeResults. That bug should be fixed in the next 
couple weeks.

Unfortunately, I don't have info about the *real* reason your query failed. I'm 
guessing it involves data skew--that is, you're performing an aggregation that 
has too many rows for one particular set of group keys. I'd suggest turning off 
"allowLargeResults" temporarily (and perhaps try adding a LIMIT to your query) 
to see if you can rustle up a different and more useful error message.

Regarding the second bug report, I'm guessing this is actually a different 
problem. If you're still seeing the error, can you send us more detail about 
exactly what API calls you're issuing that result in a 500/502? The more info 
you can provide, the better. Feel free to file a separate bug or send mail to 
jcondit@google.com.
```

Original comment by `jcon...@google.com` on 24 Mar 2014 at 3:53

**Comment by GoogleCodeExporter** [comment]
```
Closing this out due to lack of activity. Please reopen or file a new bug if 
you need additional help. (The bug mentioned in response #2 has been addressed.)
```

Original comment by `jcon...@google.com` on 16 May 2014 at 7:10
- Changed state: **Fixed**

**Comment by GoogleCodeExporter** [comment]
```
I am having the same issue today. 
```

Original comment by `franci...@plusdigitalinstitute.com` on 19 Aug 2014 at 7:15

**Comment by GoogleCodeExporter** [comment]
```
Hi Francisco,

If you are having query issues, could you please open a new issue and include

- Project id
- Job id, if you have one
- Characteristics of the problem: what happened?  What did you expect to 
happen?  Is it a recurring or one-off issue?
```

Original comment by `thomasp...@google.com` on 20 Aug 2014 at 11:18

**Comment by GoogleCodeExporter** [comment]
```
Hi Thomas, the issue was solved in the #141. Thanks
```

Original comment by `franci...@plusdigitalinstitute.com` on 22 Aug 2014 at 2:13

**Comment by GoogleCodeExporter** [comment]
```
Thank you for the confirmation!
```

Original comment by `thomasp...@google.com` on 25 Aug 2014 at 10:36
